### DISCUSSION TITLE
CVE-2023-32700

### DISCUSSION THREAD

**Discussion by MichaelCalPERS** [MiKTeX/miktex#1356] [Q&A]
Hello,
The company I work for would like to use MiKTeX. The request to use it was denied due to the following security vulnerability.

CVE-2023-32700:
LuaTeX before 1.17.0 allows execution of arbitrary shell commands when compiling a TeX file obtained from an untrusted source. This occurs because luatex-core.lua lets the original io.popen be accessed. This also affects TeX Live before 2023 r66984 and **MiKTeX before 23.5**.

Does anyone know when version 23.5 will be released and if so, will this vulnerability be addressed?

Thank you,
Michael

**Comment by wayuumunay** [comment]
Dear community I have the same concern as Michael, please help! Thanks :)

**Comment by ragibson** [comment]
23.5 was released back in May and the current Windows installer at https://miktex.org/download is version 23.9, which was released last week.
